src/quo/components/share/share_qr_code/style.cljs

```diff
@@ -71,41 +65,5 @@
    :justify-content :space-between
    :flex            1})
 
-;;; Dashed line
```

Comment:
can you please comment of why you are removing all of this?

Reply:
They were unused